doc/developer-guide.md

```diff
@@ -1125,6 +1125,16 @@ options without having to modify the source.
   behave the same way as the `CA_HOST_TARGET_<arch>_CPU` and
   `CA_HOST_TARGET_<arch>_FEATURES` CMake options and the same caveats about
   `"native"` apply here.
+* `CA_HOST_DEFERRED_COMPILATION`. This allows overriding of whether the `host`
+  device supports deferring compilation, often known as jitting. Setting to `0`
+  will disable deferred compilation, setting to `1` will enable it. Note some
+  targets such as `RISC-V` may not work fully if deferred, and is disabled by
+  default. Only available with debug support.
+* `CA_HOST_DUMP_ASM`. This supports dumping of generated assembly. Requires a
+  target which does not support deferred compilation, such as `RISC-V` or
+  setting `CA_HOST_DEFERRED_COMPILATION` to 0. This should be set to `1` or
+  unset and other values are subject to change.  Only available with debug
+  support.
```

Comment:
This reads a bit difficult, setting `CA_HOST_DEFERRED_COMPILATION` to `0` does not give you a target which does not support deferred compilation (despite our internal variable name), it gives you a target which does not *do* deferred compilation. I'm not sure I have a concrete suggestion for improvement right now though, will have another look later.

Reply:
I've tried to simplify